pkg/controller/worker/machines.go

```diff
@@ -114,23 +115,57 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 			Image:   machineImage,
 		})
 
-		volumeSize, err := worker.DiskSize(pool.Volume.Size)
+		disks := make([]map[string]interface{}, 0)
+
+		generateDiskFuc := func(volume v1alpha1.Volume) (map[string]interface{}, error) {
+			volumeSize, err := worker.DiskSize(volume.Size)
+			if err != nil {
+				return nil, err
+			}
+
+			disk := map[string]interface{}{
+				"autoDelete": true,
+				"boot":       true,
+				"sizeGb":     volumeSize,
+				"type":       volume.Type,
+				"image":      machineImage,
+				"labels": map[string]interface{}{
+					"name": w.worker.Name,
+				},
+			}
+
+			if volume.Type != nil {
+				disk["type"] = *volume.Type
+			}
+
+			return disk, nil
+		}
+
+		disk, err := generateDiskFuc(*pool.Volume)
 		if err != nil {
 			return err
 		}
 
-		disk := map[string]interface{}{
-			"autoDelete": true,
-			"boot":       true,
-			"sizeGb":     volumeSize,
-			"type":       pool.Volume.Type,
-			"image":      machineImage,
-			"labels": map[string]interface{}{
-				"name": w.worker.Name,
-			},
+		disks = append(disks, disk)
+
+		// additional volumes
+		workerConfig := &gcpapi.WorkerConfig{}
+		if pool.ProviderConfig != nil && pool.ProviderConfig.Raw != nil {
+			if _, _, err := w.Decoder().Decode(pool.ProviderConfig.Raw, nil, workerConfig); err != nil {
+				return fmt.Errorf("could not decode provider config: %+v", err)
+			}
 		}
-		if pool.Volume.Type != nil {
-			disk["type"] = *pool.Volume.Type
+		for _, volume := range pool.DataVolumes {
+			disk, err := generateDiskFuc(volume)
+			if err != nil {
+				return err
+			}
+
+			if workerConfig.Volume != nil && workerConfig.Volume.LocalSSDInterface != nil {
+				disk["interface"] = *workerConfig.Volume.LocalSSDInterface
```

Comment:
Can you add `# interface: <localSSDInterface>` to https://github.com/gardener/gardener-extension-provider-gcp/blob/master/charts/internal/machineclass/values.yaml#L10-L17 so that one can see that it is an optional value that can be specified in the Helm chart?

Reply:
Done